### ISSUE TITLE
webshop questions

### ISSUE THREAD

**Issue by ChenBinfighting1** [ZihanWang314/RAGEN#134]
Hello,
1，I would like to know wether the success rate of the webshop in the paper is a training set or a validation set?
2，How can I use StarPO-S instead of StarPO

Thanks!
